README.md

```diff
@@ -65,6 +65,9 @@ Given a path specified by `from`, upload it to the Swift container. The path mus
 
 * `from`: *Required.* A regexp specifying the file to upload.
 
+* `SegmentContainer` : *Optional* Name of the container where segments should be stored. Segments will be created for files larger than 1 GB. Default: (Container_segments)
```

Comment:
Up until now the parameters where defined in snake case. So for consistency this should be changed to `segment_container` and `segment_size`. Also maybe the default value should be written as (Default: `$container`_segments)

Reply:
should be done